lib/fluent/parser.rb

```diff
@@ -308,11 +310,16 @@ def configure(conf)
         end
 
         @time_parser = TimeParser.new(@time_format)
+
+        if @null_value_pattern
+          @null_value_pattern = /#{@null_value_pattern}/
```

Comment:
`Regexp.new` is better because users do not need to care of escaping `/` with it.